### PR TITLE
Update attirbutes paths, add region check and clusters-install-dir permissions

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by redhat-qe-bot2** [comment]
Report bugs in [Issues](https://github.com/myakove/github-webhook-server/issues)

The following are automatically added:
 * Add reviewers from OWNER file (in the root of the repository) under reviewers section.
 * Set PR size label.
 * New issue is created for the PR. (Closed when PR is merged/closed)
 * Run [pre-commit](https://pre-commit.ci/) if `.pre-commit-config.yaml` exists in the repo.

Available user actions:
 * To mark PR as WIP comment `/wip` to the PR, To remove it from the PR comment `/wip cancel` to the PR.
 * To block merging of PR comment `/hold`, To un-block merging of PR comment `/hold cancel`.
 * To mark PR as verified comment `/verified` to the PR, to un-verify comment `/verified cancel` to the PR.
        verified label removed on each new commit push.
 * To cherry pick a merged PR comment `/cherry-pick <target branch to cherry-pick to>` in the PR.
    * Multiple target branches can be cherry-picked, separated by spaces. (`/cherry-pick branch1 branch2`)
    * Cherry-pick will be started when PR is merged
 * To build and push container image command `/build-and-push-container` in the PR (tag will be the PR number).
 * To add a label by comment use `/<label name>`, to remove, use `/<label name> cancel`
 * To assign reviewers based on OWNERS file use `/assign-reviewers`
 * To check if PR can be merged use `/check-can-merge`

<details>
<summary>Supported /retest check runs</summary>

 * `/retest tox`: Retest tox
 * `/retest build-container`: Retest build-container
 * `/retest python-module-install`: Retest python-module-install

</details>

<details>
<summary>Supported labels</summary>

 * hold
 * verified
 * wip
 * lgtm

</details>

**Review by myakove — APPROVED** [review]

**Review by omrirh — APPROVED** [review]

File: openshift_cli_installer/libs/user_input.py [hunk 1]
```diff
@@ -161,6 +160,8 @@ def verify_user_input(self):
             self.assert_acm_clusters_user_input()
             self.assert_gcp_user_input()
             self.assert_cluster_platform_support_observability()
+            self.assert_missing_cluster_region()
+            self.assert_clusters_data_directory_missing_permissions()
```
Comment (myakove) [inline openshift_cli_installer/libs/user_input.py:164]: Do we need to add tests for it?

Comment (rnetser) [reply openshift_cli_installer/libs/user_input.py:164]: add `todo`

**Review by myakove — APPROVED** [review]

**Review by omrirh — APPROVED** [review]

**Review by chetna14manku — APPROVED** [review]

**Pull request merged**